docker-compose.yml

```diff
@@ -0,0 +1,8 @@
+version: "3.9"
+
+services:
+  sway:
+    build:
+      context: . 
+      dockerfile: deployment/Dockerfile
```

Comment:
Since there's no exec or cmd in the dockerfile or here, I'm not sure what this is supposed to do. Was the intention to run the language server?

Reply:
@Voxelot - is a docker-compose file needed at all for local development? I am not sure really

Reply:
This is likely a better question for @sezna